diagnostics.sh

```diff
@@ -18,7 +18,7 @@ echo "##[endgroup]"
 
 echo "##[group]failing pods"
 kubectl get pods --all-namespaces --field-selector=status.phase!=Running \
-  | tail -n +2 | awk '{print "-n", $1, $2}' | xargs -L 1 kubectl describe pod
+  | tail -n +2 | awk '{print "-n", $1, $2}' | xargs kubectl describe pod
```

Comment:
I don't think this will work when there are non-running pods in more than one namespace.  
kubectl will take the last namespace specified and ignore the earlier namespaces.  
runing one command per input line (with `-L 1`) makes more sense to me... why change it?